loopchain/utils/__init__.py

```diff
@@ -391,6 +391,34 @@ def parse_target_list(targets: str) -> list:
     return target_list
 
 
+def rename_db_dir(new_dirname: str):
+    """Rename db directory
+    This method is temporary implementation for remove 'ip:port' in db directory name
+    Ths method is useless after rename all nodes done.
+
+    :param new_dirname:
+    :return:
+    """
+    storage_path = Path(conf.DEFAULT_STORAGE_PATH)
+    if os.path.exists(storage_path / new_dirname):
+        return
+
+    regex = re.compile(r'[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}:[0-9]{4}')
```

Comment:
```suggestion
    regex = re.compile(r'[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}:[0-9]{4}')
```
or
I recommend to use string splitting and ipaddress package.

Reply:
done.